src/rpc/util.cpp

```diff
@@ -497,7 +498,7 @@ std::string RPCHelpMan::ToString() const
         if (i == 0) ret += "\nArguments:\n";
 
         // Push named argument name and description
-        sections.m_sections.emplace_back(std::to_string(i + 1) + ". " + arg.m_name, arg.ToDescriptionString());
+        sections.m_sections.emplace_back(::ToString(i + 1) + ". " + arg.m_name, arg.ToDescriptionString());
```

Comment:
I think this is the only one where `::ToString(…)` . Please change the others to simply `ToString(…)` :)